processor/tailsamplingprocessor/README.md

```diff
@@ -73,3 +79,21 @@ processors:
 
 Refer to [tail_sampling_config.yaml](./testdata/tail_sampling_config.yaml) for detailed
 examples on using the processor.
+
+### Probabilistic Sampling Processor compared to the Tail Sampling Processor with the Probabilistic policy
+
+The [probabilistic sampling processor][probabilistic_sampling_processor] and the probabilistic tail sampling processor policy work very similar:
+based upon a configurable sampling percentage they will sample a fixed ratio of received traces.
+But depending on the overall processing pipeline you should prefer using one over the other.
+
+As a rule of thumb, if you want to add probabilistic sampling and...
+
+...you are not using the tail sampling processor already: use the [probabilistic sampling processor][probabilistic_sampling_processor].
+Running the probabilistic sampling processor is more efficient than the tail sampling processor.
+The probabilistic sampling policy makes decision based upon the trace ID, so waiting until more spans have arrived will not influence its decision. 
+
+...you are already using the tail sampling processor: add the probabilistic sampling policy as last in your chain.
```

Comment:
Is adding the probabilistic sampling policy at the head of the chain considered a configuration error? If so, should a warning be generated?

Reply:
Hmm, after thinking some more this isn't actually true: the order does not matter because the tail sampling processor will always run all policies. If one of the policies returns `Sampled` it will keep the trace. I was thinking too much of this as a pipeline.